app/controllers/account_controller.rb

```diff
@@ -1,11 +1,11 @@
-class ProfileController < ApplicationController
+class AccountController < ApplicationController
   before_action :confirm_two_factor_authenticated
   layout 'card_wide'
 
   def index
```

Comment:
although I think what that is saying is that you still have a plural controller but just name the route as `get 'profile', to: 'users#show'` ?